nibabel/openers.py

```diff
@@ -9,19 +9,66 @@
 """ Context manager openers for various fileobject types
 """
 
-from os.path import splitext
-import gzip
 import bz2
```

Comment:
That looks like 2.6, and so is probably a different issue from this PR's concern. Possibly worth looking into in a new PR, but low priority in my mind. (Nobody seems to be using large .bz2 files, and who's working on HCP data with seriously old Python?)

What about 3.5?